src/key-value-engine/B-Tree-vs-Log-Structured-Merge-Tree.md

```diff
@@ -0,0 +1,130 @@
+# B-Tree vs Log-Structured Merge-Tree
+
+[B-tree](https://en.wikipedia.org/wiki/B-tree) and [Log-Structured Merge-tree](https://en.wikipedia.org/wiki/Log-structured_merge-tree) (LSM-tree) are the two most widely used data structures for data-intensive applications to organize and store data. However, each of them has its own advantages and disadvantages. This article aims to use the quantitative approaches to compare these two data structures. 
```

Comment:
Since you are saying `B-Tree and Log-Structured Merge-tree are the two` you may want a `the` on them:

```
The B-Tree and the Log-Structured Merge-tree are the two most widely...
```